test-suite/hestonslvmodel.cpp

```diff
@@ -149,9 +150,586 @@ namespace {
         f.enableExtrapolation();
         return GaussLobattoIntegral(1000, 1e-6)(f, x.front(), x.back());
     }
+
+    Real stationaryLogProbabilityFct(Real kappa, Real theta,
```

Comment:
Don't roll back these changes; but since there are still lots of files to modify, in general I'd keep helper functions closer to the tests that use them.  Incidentally, this would also reduce the diffs.

Reply:
Alright, I'll try to keep them as close as possible. 

You will also find that in some files I have combined segregated functions belonging to same namespace into a single namespace at the top. The reason for that is that when `BOOST_FIXTURE_TEST_SUITE` and `BOOST_AUTO_TEST_SUITE `is declared, it creates a new namespace. If a namespace is declared before the fixture declaration- say `namespace a`, that would act as a top-level namespace, and if the same namespace is used to add more function in it between test cases, that declares a new namespace which is `FixtureName::SuiteName::a`. To avoid conflicts and namespace errors I choose to restructure them and combine them into one before the fixture declaration.

Reply:
Oh, I see.  In that case, you're right, do move them at the top.